lib/pages/chat/chat.dart

```diff
@@ -349,6 +349,8 @@ class ChatController extends State<Chat> {
       editEvent = null;
       pendingText = '';
     });
+
+    scrollDown();
```

Comment:
I think put it in first line of the send() method have better UX because inside send() method also have await, so as a user i usually want to move right after i click send

Reply:
same idea for me